numbers.txt

```diff
@@ -1,2 +1,3 @@
 1
 2
+3
```

Comment:
Should we add a pre-commit hook that removes new lines from the end of this file to prevent the new line from accidentally being added back with a future commit?

Reply:
Yes I think that's an excellent idea, I'll leave that in your capable hands.